src/_locales/en/messages.json

```diff
@@ -179,6 +179,14 @@
         "message": "Custom API URL",
         "description": "Input label"
     },
+    "instanceDomain": {
+        "message": "Instance Domain",
+        "description": "Input label"
+    },
+    "instanceDomainPlaceholder": {
+        "message": "ex. iddqd.social",
```

Comment:
Should we reference one of the instances highlighted on the official site https://pleroma.social?